src/main/kotlin/dev/binclub/paperbin/PaperBinConfig.kt

```diff
@@ -47,6 +47,7 @@ object PaperBinConfig {
 	var antiUnicodeChat: Boolean by BooleanProperty(properties, "antiUnicodeChat", false)
 	var blockRateLimit: Boolean by BooleanProperty(properties, "blockRateLimit")
 	var chunkLoadOptimisations: Boolean by BooleanProperty(properties, "chunkLoadOptimisations", false)
+	var elytraLowerRenderDistance: Boolean by BooleanProperty(properties, "elytraLowerRenderDistance")
```

Comment:
I think it might be best to set this to false by default

Reply:
yea, thats probably a good idea